snap/snapcraft.yaml

```diff
@@ -146,6 +146,12 @@ parts:
     prime:
       - -htdocs/apps/updatenotification
 
+  # See: https://github.com/nextcloud/nextcloud-snap/pull/263
```

Comment:
Oh yes! Can we actually remove the link to the PR now that we have a nice explanation there?